.github/workflows/CI.yml

```
@@ -28,6 +26,7 @@ jobs:
           repository: 'breez/breez-sdk'
           ssh-key: ${{secrets.REPO_SSH_KEY}}
           path: 'breez-sdk'
+          ref: ${{ inputs.ref }}
```

There won't be an input on `push` or `pull request`, so perhaps add a fallback to the github sha to make sure the field is always set.

```suggestion
          ref: ${{ inputs.ref || github.sha }}
```